src/it/scala/io/scalac/tezos/translator/LoginRoutesSpec.scala

```diff
@@ -28,7 +28,7 @@ class LoginRoutesSpec
     val loginRoute: Route = new LoginRoutes(new UserService(new UserRepository, testDb), system.log).routes
 
     "LoginRoute" should "reject wrong credentials" in {
-      Post("/login", UserCredentials("admin", "asdf")) ~> loginRoute ~> check {
+      Post("/login", UserCredentials("admin", "admin")) ~> loginRoute ~> check {
```

Comment:
why ?

Reply:
misstake. I just replay all login which was asdf, Everywhere except this place password was zxcv . Resolved